fast64_internal/oot/scene/operators.py

```diff
@@ -163,7 +163,9 @@ def execute(self, context):
                 exportInfo = ExportInfo(True, bpy.path.abspath(settings.exportPath), None, levelName)
             else:
                 if option == "Custom":
-                    subfolder = "assets/scenes/" + settings.subFolder + "/"
+                    subfolder = (
+                        context.scene.fast64.oot.get_extracted_path() + "assets/scenes/" + settings.subFolder + "/"
+                    )
```

Comment:
Change to the f-string approach shown in the next change below?

Reply:
it's a bit weird to use f-strings with the function as the value returned ends with ``/``

though it's fine for now, I think we should use pathlib more in general anyway so it can be addressed in a general cleanup (though I'll f-string the other parts)

Reply:
I fixed the slash oddities but I didn't use f-strings eveywhere because of the line length limitation of the formatter